### PR TITLE
Added blurb about LocationComponent elevation `0f` parameter

### LINKED ISSUE #938: Document that LocationComponentOptions.elevation(0) turns off the shadow
Would be good to add a mention of elevation. Based on @zestyping 's pr at https://github.com/mapbox/mapbox-gl-native/pull/14471

### PR DESCRIPTION
**Pull request by langsmith**
This pr resolves #938 by adding a blurb about how passing `0f` for the `LocationComponent` `LocationComponentOptions#elevation()` method parameter will remove the shadow underneath the `LocationComponent`.

Other deletion in this pr is removing a duplicate styling option section, which was left over from past shuffling of content on this page.

Related: https://github.com/mapbox/mapbox-gl-native/pull/14471

### CODE REVIEW

**Comment by langsmith** [comment]
cc @zestyping , who originally brought this up in `/mapbox-gl-native`

**Comment by langsmith** [comment]
Screen Shot 2019-04-23 at 5 06 29 PM

File: src/pages/maps/overview/location-component.md [hunk 1]
```diff
@@ -95,7 +95,7 @@ locationComponent.activateLocationComponent(locationComponentActivationOptions)
 | mapbox_navigationDrawable | Drawable image used for the navigation state icon  |
 | mapbox_accuracyAlpha | The larger light blue circle surrounding the device location icon   |
 | mapbox_accuracyColor | Color of the larger light blue circle surrounding the device location icon   |
-| mapbox_elevation | The amount of space between the map and the device location icon. The elevation will adjust the size of the shadow as seen underneath the white circle   |
+| mapbox_elevation | The amount of space between the map and the device location icon. The elevation will adjust the size of the shadow as seen underneath the white circle. Passing `0f` as the method parameter will remove the shadow effect.  |
```
Comment (colleenmcginnis) [inline src/pages/maps/overview/location-component.md]: Actually one small comment that may only bother me: if one of the rows in this able ends in `.` all of them should end in `.`.

Comment (langsmith) [reply src/pages/maps/overview/location-component.md]: `.` added to all of them

**Pull request merged**